wicker/core/datasets.py

```diff
@@ -22,6 +24,154 @@
 FILE_LOCK_TIMEOUT_SECONDS = 300
 
 
+def get_file_size_s3_proced(buckets_keys: List[Tuple[int, int]]) -> int:
```

Comment:
I might suggest the function name `get_file_size_s3_multiproc` or `get_file_size_s3_multiprocess` but feel free to ignore and mark this comment as resolved.

Reply:
I like multiproc better, proced sounds weird thinking about it.